src/LandingPage.tsx

```
@@ -0,0 +1,462 @@
+import React, { useState, useEffect } from 'react';
+import { motion, useScroll, useTransform } from 'motion/react';
+import { Button } from '@/components/ui/button';
+import { Card, CardContent } from '@/components/ui/card';
```

Biome (2.1.2)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Unsafe fix: Remove the unused imports.


(lint/correctness/noUnusedImports)